packages/js/src/components/SEMrushRelatedKeyphrasesModalContent.js

```diff
@@ -117,14 +118,14 @@ export default function RelatedKeyphraseModalContent( props ) {
 			) }
 
 			{ getUserMessage( props ) }
-
-			<SEMrushKeyphrasesTable
-				keyphrase={ keyphrase }
-				relatedKeyphrases={ relatedKeyphrases }
-				countryCode={ countryCode }
-				renderAction={ renderAction }
-				data={ response }
-			/>
+			<Root>
+				<KeyphrasesTable
+					relatedKeyphrases={ relatedKeyphrases }
+					columnNames={ response?.results?.columnNames }
+					data={ response?.results?.rows }
```

Comment:
Can they not exist? In particular the columnNames. Should we still render if that is the case?